.github/workflows/tests.yml

```diff
@@ -65,7 +85,7 @@ jobs:
     if: github.event_name == 'pull_request' && needs.tap_syntax.outputs.run-tests
     strategy:
       matrix:
-        version: ['11.0', '10.15', '10.14']
+        version: ['11-arm', '11.0', '10.15', '10.14']
```

Comment:
Sorry to be a pain but: could this be updated to `11-arm64` on these workers and, even better still, use `11` instead of `11.0` for Big Sur at some point too (this could be separate tasks and I can open an issue for tracking if easier). This will allow simplifying some code in Homebrew/brew.

Reply:
Yeah we need these to be homogeneous at some point, but it's not gonna happen until we are updating the nodes, because it's too much of a pain to do

Reply:
Yeh, no rush but when the nodes are next updated it'd be much appreciated if you could handle this for the ARM ones (and @Rylan12 @jonchang @Bo98 or whomever handles the Big Sur ones).

Reply:
(it'd be OK to apply both labels for now so code changes aren't needed to have it keep working)

Reply:
Agreed. Our runners are already running 11.1 so the names are already out of date 😅. Also agree that this doesn't need to happen immediately. Might be good to get some PRs open to get ready for the next time the nodes are updated.

In terms of things that need to be modified in the brew repo, it looks like it's pretty much just `dispatch-build-bottle`, right? Looks like the only part that would need to change is [the label mapping](https://github.com/Homebrew/brew/blob/5818b284797cb470d49b0d63edaffd5283e3dc2f/Library/Homebrew/dev-cmd/dispatch-build-bottle.rb#L50-L56). And then, once this is complete, [this mapping](https://github.com/Homebrew/brew/blob/5818b284797cb470d49b0d63edaffd5283e3dc2f/Library/Homebrew/dev-cmd/dispatch-build-bottle.rb#L38-L40) can be removed as well.

I won't be able to do the actual renaming as I'm not an organization owner. Happy to help out with the `dispatch-build-bottle` and whatever other code changes are needed, though.